### PR TITLE
Jasmine 2.5.2

### PR DESCRIPTION
**Pull request by andyholmes**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 2.5.2 - August September 4, 2020
+
+- The internal copy of the Jasmine library has been updated to version 2.5.2, which brings:
+  - Add `toBeGreaterThanOrEqual()` and `toBeLessThanOrEqual()` matchers
+  - Rename `j$` to `jasmineUnderTest` for specs
```
Comment (ptomato) [inline NEWS.md]: This isn't part of the API so we can probably leave it out.

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 2.5.2 - August September 4, 2020
```
Comment (ptomato) [inline NEWS.md]: September :-)

**Review by ptomato — APPROVED** [review]
Thanks! I'm happy to make those changes if I get around to it before you do.

File: NEWS.md (hunk 2 above)
Comment (andyholmes) [reply NEWS.md]: :facepalm:

**Review by ptomato — APPROVED** [review]
Thanks!

**Pull request merged**